DoSomething/gladiator
# Add more tokens

#### What's this PR do?

* Added more tokens that can be used in emails:
```
            :end_date:
            :leaderboard_msg_day:
            :leaderboard_msg_day-1: 
            :first_name:           
            :sender_name:          
            :campaign_title:        
            :reportback_noun:     
            :reportback_verb:
```

**OTHER UPDATES**
* Only sends one test message to the contest sender on test environments
* Updated the leaderboard_msg_day accessor so that it just returns and `int` we can use to do that leaderboard msg logic with. 
* Moved the processing of messages into the queue handle to help with performance.

#### How should this be manually tested?

pull down branch

make sure `APP_DEBUG=true` 

Add the tokens in this PR to an email and send

You should get an email with the tokens replaced.

#### What are the relevant tickets?
Addresses #153
